From 179e1429091b9a0b647acdff667b8b40c86f8f6e Mon Sep 17 00:00:00 2001
From: Seokha Ko <sh.ko@lge.com>
Date: Sun, 30 Apr 2017 14:39:39 +0900
Subject: [PATCH 1/2] allow setting sysroot for libgcc lookup

:Release Notes:
allow setting sysroot for libgcc lookup

:Detailed Notes:
allow setting sysroot for libgcc lookup

:Testing Performed:

:QA Notes:

:Issues Addressed:
---
 Makefile | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/Makefile b/Makefile
index ea7fd000ad..c780cff3b1 100644
--- a/Makefile
+++ b/Makefile
@@ -684,7 +684,7 @@ u-boot-main := $(libs-y)
 ifeq ($(CONFIG_USE_PRIVATE_LIBGCC),y)
 PLATFORM_LIBGCC = arch/$(ARCH)/lib/lib.a
 else
-PLATFORM_LIBGCC := -L $(shell dirname `$(CC) $(c_flags) -print-libgcc-file-name`) -lgcc
+PLATFORM_LIBGCC := -L $(shell dirname `$(CC) $(c_flags) $(LIBGCC_LOCATE_CFLAGS) -print-libgcc-file-name`) -lgcc
 endif
 PLATFORM_LIBS += $(PLATFORM_LIBGCC)
 export PLATFORM_LIBS
-- 
2.11.0

